### PR TITLE
Console - application general migration - seventh part

### PR DESCRIPTION
**Pull request by jansobczynski**
## Issue

https://gravitee.atlassian.net/browse/APIM-XXX

## Description

Migrating application groups tab

## Additional context
Screenshot:
Screenshot 2023-12-14 at 2 09 26 PM

🏗️ Your changes can be tested here and will be available soon:
      Console: [https://pr.team-apim.gravitee.dev/6207/console](https://pr.team-apim.gravitee.dev/6207/console)
      Portal: [https://pr.team-apim.gravitee.dev/6207/portal](https://pr.team-apim.gravitee.dev/6207/portal)
      Management-api: [https://pr.team-apim.gravitee.dev/6207/api/management](https://pr.team-apim.gravitee.dev/6207/api/management)
      Gateway v4: [https://pr.team-apim.gravitee.dev/6207](https://pr.team-apim.gravitee.dev/6207)
      Gateway v3: [https://pr.gateway-v3.team-apim.gravitee.dev/6207](https://pr.gateway-v3.team-apim.gravitee.dev/6207)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-alcjhkovqw.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/services-ngx/application-user-and-group.service.ts
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/services-ngx/application-user-and-group.service.ts:1]: Not needed right ?

Comment (jansobczynski) [reply gravitee-apim-console-webui/src/services-ngx/application-user-and-group.service.ts:1]: unused method deleted

Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/services-ngx/application-user-and-group.service.ts:1]: hum ok. 
But like api service (and one of my previous comment) could you split it into dedicated business :
- `application-memebers.service.ts` 
- and use getGroups from `GroupService`

Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/services-ngx/application-user-and-group.service.ts:1]: ho and we already have ApplicationMembersService. so this service it's not needed

**Review by ThibaudAV — CHANGES_REQUESTED** [review]
To change : Remove duplicated service method in ApplicationUserAndGroupService and use the one in `ApplicationMembersService` &  `GroupService` instead

**Review by ThibaudAV — APPROVED** [review]

File: gravitee-apim-console-webui/src/services-ngx/application-user-and-group.service.ts
Comment (jansobczynski) [reply gravitee-apim-console-webui/src/services-ngx/application-user-and-group.service.ts:1]: done

**Pull request merged**